count-things/javascript/analysis.js

```diff
@@ -0,0 +1,14 @@
+import { createJsVirDir } from './create-js-vir-dir.js';
+import { analyzeVirDir } from './analyze-vir-dir.js';
```

Comment:
remind me what `vir` means here?

Reply:
virtual directory.  a an object that mirrors the directory you want to analyze along with other relevant meta-info

Reply:
thought so...it might be more obvious to rename it at some point to avoid confusion with some sort of containerized directory, or VM-environment. Maybe something referring to the data being placed entirely in memory or something. Anyway, not a big deal.